client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -96,7 +117,7 @@ final case class Client(open: Service[Request, DisposableResponse], shutdown: Ta
     }
   }
 
-  @deprecated("Use fetchAs", "0.12")
+  @deprecated("Use expect", "0.14")
```

Comment:
I've been surprised by how many of our users run older versions, and the compiler makes a nice migration tool.  I'm in favor of aggressively reaping anything that claims valuable namespace or causes architectural drag, and less hurried on cases like this.